### PR TITLE
Feat: Improved and fixed style

### PR DESCRIPTION
**Pull request by LeoBruant**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Closes #(issue) | No issue linked.

## Type of change

Please add any relevant information to the type of change you've made.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (or is a documentation update)
- [ ] HIGH IMPORTANCE (requires a fast merging)

## Has This Been Tested?

Please describe the tests that you ran to verify your changes. It is way more likely for a PR to be merged and approved with successful tests.

- [ ] Yes, I ran tests (describe below)
- [x] No, I did not include any test
- [ ] No test is needed for this change

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

## Browser compatibility (optional)

Which browser was this tested on?

- [ ] Chrome
- [ ] Firefox
- [ ] Opera
- [ ] Safari
- [ ] Edge
- [ ] Samsung Internet
- [ ] Others (specify)

### CODE REVIEW

File: templates/components/table/row/actions.html.twig [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
-      </a>
-    {% endif %}
-  {% endfor %}
-</td>
\ No newline at end of file
+<td class="flex gap-2">
+    {% for key, action in config.actions %}
+        {% if action.validation is defined %}
+            <label data-action="click->table#action"
+                for="action-modal"
```
Comment (YummYume) [inline templates/components/table/row/actions.html.twig:5]: Le `for` vise toujours la même modal c'est normal?

File: templates/components/table/row/actions.html.twig [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
+                class="btn-{{ key }} btn-square btn"
+                data-route="{{ path(action.route, { id: config.value }) }}"
+                data-message="{{ action.validation.message|trans({}, "table+intl-icu") }}">
+                {% include "icon/#{action.icon}.html.twig" %}
+            </label>
+        {% else %}
+            <a href="{{ path(action.route, { id: config.value }) }}">{{ component("button", { icon: action.icon, square: true, variant: key }) }}</a>
```
Comment (YummYume) [inline templates/components/table/row/actions.html.twig]:
```suggestion
            <a href="{{ path(action.route, { id: config.value }) }}">{{ component("button", { icon: action.icon, square: true, variant: key, content: '...' }) }}</a>
```

Ne jamais mettre de lien/bouton sans texte. Mettre `aria-label` si c'est un icon.

File: templates/admin/user/index.html.twig [hunk 3]
```diff
@@ -4,14 +4,21 @@
     Admin - User
 {% endblock %}
 
+{% block main_title %}
+    Admin - User
```
Comment (YummYume) [inline templates/admin/user/index.html.twig]:
```suggestion
    Admin - User
```

Traduction + meta description

File: templates/admin/homepage/index.html.twig [hunk 4]
```diff
@@ -10,13 +10,14 @@
...
-    <h1>
-        SUPER SECURE PAGE
-    </h1>
     <div class="flex">
-        {{ knp_menu_render('backoffice', { template: knp_custom_menu }) }}
+        {{ knp_menu_render("backoffice", { template: knp_custom_menu }) }}
     </div>
 
-    <a class="btn" href="{{ path("security_logout") }}">{{ "common.logout"|trans }}</a>
+    <a href="{{ path("security_logout") }}">{{ component("button", { content: "common.logout"|trans }) }}</a>
```
Comment (YummYume) [inline templates/admin/homepage/index.html.twig]: Je pense qu'on peut retirer j'avais juste mis le bouton en attendant

**Review by YummYume — CHANGES_REQUESTED** [review]

File: templates/components/table/row/actions.html.twig (hunk 1 above)
Comment (Spacelocust) [reply templates/components/table/row/actions.html.twig:5]: Ouais c'est normal, j'ai fait en sorte que mon système utilise une seule modal pour pas en avoir un nombre trop élevé en fonction des lignes du tableau

**Review by YummYume — APPROVED** [review]

**Pull request merged**